packages/react/src/themes/teams/components/Dropdown/dropdownVariables.ts

```diff
@@ -30,12 +30,12 @@ export default (siteVars): DropdownVariables => ({
   borderWidth: `0 0 ${pxToRem(2)} 0`,
   color: siteVars.bodyColor,
   comboboxPaddingButton: `0 ${_12px_asRem}`,
-  comboboxFlexBasis: '50px',
+  comboboxFlexBasis: pxToRem(50),
   listBackgroundColor: siteVars.colors.white,
   listBorderRadius: _3px_asRem,
   listPadding: `${pxToRem(8)} 0`,
   listBoxShadow: `0 .2rem .6rem 0 ${siteVars.gray06}`,
-  listMaxHeight: '20rem',
+  listMaxHeight: pxToRem(296),
```

Comment:
just curious how 20rem became 296px?

Reply:
So previously on the Dropdown list, we had padding on the top and 6 elements in the list (one list item has height of 48px) => (6*48px) + 8px = 296px :) We don't want to use rem anyway and the height was a bit different because of the `box-sizing`, so that is why I converted it to px.